projects/packages/licensing/src/class-endpoints.php

```diff
@@ -186,23 +185,6 @@ public static function set_jetpack_license_key_permission_check() {
 		return new WP_Error( 'invalid_user_permission_set_jetpack_license_key', self::$user_permissions_error_msg, array( 'status' => rest_authorization_required_code() ) );
 	}
 
-	/**
-	 * Verify that user can view and update user-licensing data.
-	 *
-	 * @since 1.7.0
-	 *
-	 * @return bool Whether the user is currently connected and they are the connection owner.
-	 */
-	public static function user_licensing_permission_check() {
-		$connection_manager = new Connection_Manager( 'jetpack' );
-
-		if ( $connection_manager->is_user_connected() && $connection_manager->is_connection_owner() ) {
-			return true;
-		}
-
-		return new WP_Error( 'invalid_permission_manage_user_licenses', self::$user_permissions_error_msg, array( 'status' => rest_authorization_required_code() ) );
```

Comment:
Removal of this makes the code here unused and may be we should have cleaned that up.

https://github.com/Automattic/jetpack/blob/c37d2b4be2dcff6cc5032a1ad18cb3562dc67df7/projects/js-packages/licensing/components/activation-screen/index.jsx#L113

Reply:
I missed this. Agreed; @gmjuhasz any chance you can do a cleanup PR to remove the unused code?

Reply:
Created here https://github.com/Automattic/jetpack/pull/44654